app/json/slate_config.schema.json

```diff
@@ -145,6 +145,9 @@
                   "top15",
                   "top30",
                   "top45",
+                  "thompson-sampling-1day",
+                  "thompson-sampling-7day",
+                  "thompson-sampling-14day",
                   "thompson-sampling"
```

Comment:
I'd like this change to `thompson-sampling-28day` rather than being implicit

Reply:
this change is only failing 79 of the 95 tests so far